hpc-software.md

```diff
@@ -0,0 +1,149 @@
+> the same modules as part of your job script before running the main command. 
+> 
+> You can also load modules to run specific software. If done for interactive 
+> testing, this should be done in an interactive job; otherwise, the module 
+> should be loaded in the job submit file. 
+
+## C. Unload Software in Modules
+
+If you no longer want to use a specific software installation, you can "unload"
+the software module with the following command: 
+
+```
+[alice@login]$ module unload module_name
+```
+
+If you want to clear your command line environment and start over, run the following: 
+
+```
+[alice@login]$ module purge
+```
+
+# 3. Installing Software on the Cluster
+
+## Overview
+
+Unless you are using a licensed software program provided via modules, you 
+are able to compile and install the software you need on the HPC Cluster. 
+Software should be installed to your `/software/username` 
+directory. If using CHTC's provided compilation tools via modules, make 
+sure to load the needed modules before compiling and to load the same 
+modules in your job submission. 
+
+For groups that would like to share software installations among group
+members, please contact us about getting a shared "group" directory. 
+
+If you are new to software installation, see the section below for 
+a more step-by-step description of the process. 
+
+## Step by Step Process
+
+1. **Download Source Code** - download the source code for your desired program to 
+	your `/software` directory.
+1. **Read the Docs** - try to find the installation instructions, either online or 
+	in the downloaded source code. In particular, you'll want to note if there are 
+	any special requirements for dependencies like MPI or the compiler needed. 
+1. **Load Modules** - if you are using software modules to help you build your 
+	code, load them now. Keep track of what you use so that you can load them 
+	in your job submit file later. We also recommend doing a `module purge` before 
+	loading your modules to make sure you're starting from a clean environment. 
+1. **Install** - most scientific software follows the three step installation process
+	of `configure` - `make` - `make install`. 
+	1. `configure`- this step checks for tools and requirements needed to compile 
+		the code. This is the step where you set the final installation location of 
+		a program. The option for setting this location is typically called the 
+		"prefix"; a common syntax is: 
+			~~~
+			$ ./configure --prefix=/software/user
+			~~~
+		This is where you will want to set the installation location to be your 
+		`/software` directory. 
+	1. `make` - this step compiles and links the code, turning it from human-readable 
+		source code to compiled binary code. This is usually the most time consuming 
+		step of the installation process. 
+	1. `make install` - this step copies compiled files to the final installation location 
+		(usually specified in the `configure` step). 
+
```

Comment:
Consider adding a short section here about cleaning up build directories.  Some things you might save from an Autoconf-iscated software install are the stdout, stderr, and the `config.log` written out during the `./configure` step.  The rest is usually never used again after `make install`.